modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/actions/ActionsDropdown.js

```diff
@@ -146,7 +146,7 @@ function ActionsDropdown({
 
 		const onActionDropdownItemClick = context.onActionDropdownItemClick;
 
-		const url = formatActionURL(action.href, itemData);
+		const url = action.href ? formatActionURL(action.href, itemData) : "";
```

Comment:
why you don't push this logic into formatActionURL?

Reply:
Because I am not sure whether this logic is correct inside the `formatActionURL`? We know that empty the URL in this file will fix the issue, we do not know if empty the URL in formatActionURL will cause any other regression bugs because there are other functions using it.